tests/test_main.py

```diff
@@ -7,7 +7,7 @@
 
 import pytest
 
-from credsweeper import __main__, ByteContentProvider, StringContentProvider
+from credsweeper import __main__, ByteContentProvider, StringContentProvider, file_handler
```

Comment:
It would be good to change with below code.

``` python
from credsweeper.file_handler.byte_content_provider import ByteContentProvider
```

It isn't match with `TextContentProvider`. Please check.

Reply:
Or you can merge both of them like this.

``` python
from credsweeper.file_handler import ByteContentProvider, TextContentProvider
```

Reply:
Ok. The unit test is not related for fuzzing - i'll move it to other PR